### ISSUE TITLE
Farmer Loan Management System

### ISSUE THREAD

**Issue by sambhav2612** [JoinColony/colonyHackathon#84]
### Colony Hackathon Submission

**Farmer Loan Management System**

---

**Project Description**

In India, farming, in general, is not a bliss for an average farmer. Being difficult and convoluted at times, farmers usually take heavy loans from the rich people (who are generally involved in illegal activities but not necessarily) in their demographics which when not repaid leads to heavy loss of life, either forcefully or under social pressure. Since this is a pretty catastrophic step, the first thing our project should intend to do is guide the farmers to consult with their regional legislative leader about the schemes that the Indian Government has put into motion which provides a good amount of money for their loans on more reasonable rate than the people involved earlier. Hence, avoiding chances of acquainting with the former people *et al*. This puts the ***reputation*** of the regional leader at stake.

*Therefore, the lenders can belong to two domains: Regional People or Indian Government.*
 
For first domain, a good thought could be put into managing this [sort of a business] model so as to predict the outcome of such an event, depending on the crop being planted and other environmental factors related to the crop.

The management can come in terms of the ***decision making, dispute resolution, and financial management*** procedures empowered through a Colony. Whenever an agreement is called upon after monitoring the crop season, both parties (lender and taker) become part of a separate Colony which governs some mutual rules of understanding for dispute resolution and finance management [most likely] on a per-month basis if the lender party comes from the first domain, which when violated drops their reputation by some factor of their previous contributions to the agreement. The same goes for when the agreement is not violated leading to an increase in each parties' reputation among their social circles. 

Else, the entire thing could be managed and monitored by the government itself.

**Project Repository**

https://github.com/sambhav2612/flms

**Team Members and Contact info**

Sambhav Jain (goes by @sambhav2612 everywhere)

---

Also, we're looking for other community members to come onboard, share their thoughts, refine the final work product and help us in shaping this into reality,

**NOTE:** *Team members would be selected based on their previous experiences with the Indian Farming Culture and loaned-farming in general. However, refinements would be made based on the suggestions from **every conversation participant** of the project*

**FOR PARTICIPANTS HAVING NO KNOWLEDGE TO COLONY**: Please get yourself acquainted at https://joincolony.github.io/colonynetwork/docs-colony/ before involving in conversation, thank you!

**Comment by sambhav2612** [comment]
@jrmoreau please add the appropriate tags to help notify other participants of this idea.

**Comment by gichiba** [comment]
@sambhav2612 You can also add the tags! they're right up there on the right hand side! 

I think this is a fantastic use case! I only know a little about farming and microfinance from time I spent in Myanmar, but I know that it's an extremely valuable tool to have even a little bit of access to loans of a reasonable interest rate. 

One way I'd be curious to know is how to map domains and tasks to the actual farming activities. In microfinance, the practice of 'group lending' is often used to better incentivize a community of farmers to keep current on their payments. Perhaps you could use the domains in colony to organize groups of farmers with outstanding loans, and map each loan repayment to a task. (this is just a first idea, I'm sure there are other, maybe better ways to do that). 

Will you be organizing this project on your own chatroom, or talking in this gitter, or somewhere else?

**Comment by arealdeadone** [comment]
@sambhav2612  yeah, definitely I think these two can be merged. Let's get in touch and discuss it further. Contact me at ari0997@gmail.com

**Comment by zillerium** [comment]
what is the status now?

**Comment by jrmoreau** [comment]
@sambhav2612 @arealdeadone is there room for additional members?

**Comment by ryanchristo** [comment]
@sambhav2612 @arealdeadone Have you officially merged? If so, which project will represent the merged project moving forward? Are you accepting new team members?

I have experience with React, Redux, Webpack, Node and working knowledge of Web3 and Solidity. Looking through projects and Colony documentation today with a lot of time in the coming week to help out. Let me know if you are interested in working together.

**Comment by gichiba** [comment]
Hey Ryan, they have I think combined and formed a slack to coordinate, I think you'll need to reach out to @arealdeadone for an email invite.
